openipmi.yaml

```diff
@@ -1,10 +1,10 @@
 package:
   name: openipmi
   version: 2.0.34
-  epoch: 0
+  epoch: 1
```

Comment:
It seems this package doesn't build anymore?

```
2024/05/15 16:21:41 WARN configure: error: libtermcap, libcurses or libncurses are required!
```

Reply:
@imjasonh ncurses build failure is resolved by making sure dev libraries of curses link with tinfo.